pages/send/Send.tsx

```diff
@@ -21,6 +22,48 @@ export function Send() {
   const [isLoading, setLoading] = React.useState(false);
   const [startScanning, setStartScanning] = React.useState(false);
 
+  async function pickImage() {
```

Comment:
Clicking on image icon currently crashes on iOS (yarn device:ios), looking into it!

Reply:
Resolved by adding the permission to app.config.js